Monsoon/HVPM.py

```diff
@@ -78,7 +78,7 @@ def amps_from_raw(self,raw):
         return result
 
     def raw_from_amps(self,value):
-        result = 65535 - 0x0F00 * (value / 65535) + 0x0F00
+        result = (65535 * value / 15.625)
```

Comment:
Empirically this was acting about right.  It might still be wrong, can you confirm?  The documentation is a little confusing.